src/main/java/models/SpanishGame.java

```diff
@@ -0,0 +1,31 @@
+package models;
+
+import java.util.ArrayList;
+
+/**
+ * Created by harderg on 2/14/16.
+ */
+public class SpanishGame extends Game {
+
+    /*public void buildDeck() {
+        for(int i = 1; i < 14; i++){
+            if (i < 8 || i > 10) {
+                deck.add(new Card(i, SpanishSuit.Clubs));
+                deck.add(new Card(i, SpanishSuit.Coins));
+                deck.add(new Card(i, SpanishSuit.Swords));
+                deck.add(new Card(i, SpanishSuit.Cups));
+            }
+        }
+    }*/
+
+    public void buildDeck() {
```

Comment:
@mihaidan I have in my notes from our group meeting that what was unique to the game classes are: how the cards are built, how the deck is built, and how the remove works. On a closer look the deck and cards are both built in this function. As far as remove goes, by changing this loop to go from 1-14 instead of 2-15 the aces are now low, so the remove does not need to be changed.
Thanks for the double check though!